riscv/riscv.mk.in

```diff
@@ -1260,6 +1260,15 @@ riscv_insn_svinval = \
 	hinval_vvma \
 	hinval_gvma \
 
+riscv_insn_cmo = \
```

Comment:
This is not sail-riscv but actual extension name would be better here, too. How about `zicbo` (for `Zicbo[mpz]`)?

Reply:
The name in riscv_insn_list are all use the total name of extensions(k, b, v,...) or specific extension name(svinval). I think cmo is more like the total name of zicbo[mpz] than zicbo.

Reply:
Got it. If the maintainer agrees, that's okay.